docs/4.0/components/forms.md

```diff
@@ -1171,6 +1171,20 @@ As is the `size` attribute:
 </select>
 {% endexample %}
 
+### Range
+
+Create custom `<input type="range">` controls with `.custom-range`. The track (the background) and thumb (the value) are both styled to appear the same across browsers. As only IE and Firefox support "filling" their track from the left or right of the thumb as a means to visually indicate progress, we do not currently support it.
+
+{% example html %}
+<input type="range" class="custom-range">
+{% endexample %}
+
+Additionally, you may specify steps by adding `min` and `max` attributes.
```

Comment:
To be a bit more specific: if you don't add a `min`/`max` to a range input, it defaults to `min=0` and `max=100`. if you don't add a specific `step`, the step value is 1.

So you're not really "specifying steps" by adding `min` and `max`. You're going from a default of 101 steps to a more specific number of steps. you could set `min=0 max=5 step=0.05` and have 101 steps again.

Reply:
also, for both examples, you should really have either a `<label for="...">` or `aria-label` or similar...

Reply:
Noted. I'll take a pass at updating the docs.